exercises/practice/linked-list/.docs/instructions.append.md

```diff
@@ -0,0 +1,3 @@
+# Introduction append
+
+In immutable, strict languages such as Elixir, there is no way to create cyclical data structures, like doubly linked lists, since they require mutability. You must therefore simulate that behavior in a different way.
```

Comment:
This could use a citation if we wanted, this would be a potential: https://elixirforum.com/t/cyclical-data-structures-in-elixir/2987/6